markovbot/commands.py

```diff
@@ -44,3 +47,71 @@
     await markovbot.say(sentence)
 
 
+@markovbot.command(pass_context=True, help='Mock the last specified user message.')
+async def mock(context, user: str = None):
+    server_context = context.server_context
+
+    if not server_context.is_ready:
+        await markovbot.say('I am still learning from all your messages. Try again later.')
+        return
+
+    if user is None:
+        log.debug('Bot issued say command on server %s',
+                  server_context.server.name)
+        sentence = server_context.markov.make_sentence_server()
+
+        if sentence is None:
+            await markovbot.say('Unable to generate message. This is probably due to a lack of messages on the server.')
+            return
+
+    else:
+        logs = markovbot.logs_from(server_context.message.channel, limit=20)
+        sentence = logs[-1]
+
+        sentence = mockString(sentence)
+        
+    await markovbot.say(sentence)
+
+
+def mockString(sentence):
```

Comment:
NIT: Make the function signature:
def mock_string(sentence: str): str

Just to show that it takes a string and returns a string.

Reply:
Added.